snooty.toml

```diff
@@ -15,3 +15,13 @@ csharp-docs-versioned = "https://mongodb.github.io/mongo-csharp-driver/{+csharp-
 
 java-version = "4.8"
 java-api = "https://mongodb.github.io/mongo-java-driver/{+java-version+}"
+
+[[banners]]
+targets = ["csharp.txt"]
+
+variant = "tip"
+value = """ \
+    The new .Net/C# driver documentation is now live. To learn more about using \
+    the driver in your application, see the `MongoDB C# Driver Documentation \
```

Comment:
Suggestion:
Use the "official MongoDB ... driver" somewhere (whether in the link or the prior instance) to identify it as official.